config/dev/manager_dev.yaml

```diff
@@ -24,6 +24,7 @@ spec:
             - -zap-devel=true
             - -enable-controller-kongplugininstallation=true
             - -enable-validating-webhook=true
+            - -enable-controller-konnect=true
```

Comment:
Please add this to 
- `config/debug/manager_debug.yaml` (btw. this `=true` next to each flag seems redundant)
- [Makefile](https://github.com/Kong/gateway-operator/blob/ebb8b70cd7548f15cbf165898c6510e75afb79ac/Makefile#L471)

somewhere else? (idk)